scripts/pre_commit_linter.py

```diff
@@ -2075,93 +2135,36 @@ def _check_codeowner_file(self):
+            for file_paths in _walk_with_gitignore('.', dir_patterns):
+                for file_path in file_paths:
+                    match = False
+                    for file_pattern in file_patterns:
+                        if file_path in glob.glob(file_pattern):
                             match = True
-                            continue
-                        for path_to_match in path_patterns:
-                            # The level of the file in the directory
-                            # structure. For e.g. /core/controllers/
-                            # domain.py is on third level.
-                            # This condition checks if the path to check
-                            # is a directory or a file. If it is a
-                            # file, the level would be the same as found
-                            # by calculating len(path_to_match.split('/'))
-                            # but is reduced by one if it is a directory
-                            # since the split command will return an empty
-                            # string at the last of the list which will
-                            # wrongfully increase the level.
-                            if path_to_match.split('/')[-1]:
-                                level = len(path_to_match.split('/'))
-                            else:
-                                level = len(path_to_match.split('/')) - 1
-                            # This condition finally matches the file being
-                            # walked currently against the path from the
-                            # CODEOWNERS file. The level helps in matching
-                            # by considering the file name upto only the
-                            # the level of the CODEOWNERS path. For e.g.
-                            # if the file being walked upon is /core/domain
-                            # /domain.py and the path from CODEOWNERS to be
-                            # matched is /core/ then it will only consider
-                            # the file name upto level 1 i.e. just the '/core'
-                            # part of the file name since it is sufficient to
-                            # be matched.
-                            if os.path.join(*((
-                                    os.path.join(root, file_name).replace(
-                                        './', '', 1)).split(
-                                            '/')[:level])) in glob.glob(
-                                                path_to_match):
-                                match = True
-                                break
-                        if not match and self.verbose_mode_enabled:
-                            print ('WARNING! %s/%s is not covered under '
-                                   'CODEOWNERS' % (root, file_name))
+                            break
+                    if not match and self.verbose_mode_enabled:
```

Comment:
If verbose mode is not enabled, we still need to show the viewer which file failed, right? Shouldn't that be stored in a summary message somewhere?

Reply:
Ah, I've done my whole testing in `--verbose` mode, Fixed!